mindsdb/integrations/handlers/intercom_handler/intercom_handler.py

```diff
@@ -0,0 +1,130 @@
+from mindsdb.integrations.handlers.intercom_handler.intercom_tables import Articles
+from mindsdb.integrations.libs.api_handler import APIHandler
+from mindsdb.integrations.libs.response import HandlerStatusResponse as StatusResponse
+from mindsdb_sql import parse_sql
+import requests
+import pandas as pd
+from collections import OrderedDict
+from mindsdb.integrations.libs.const import HANDLER_CONNECTION_ARG_TYPE as ARG_TYPE
+import json
+from mindsdb.utilities import log
+
+
+class IntercomHandler(APIHandler):
+    def __init__(self, name: str, **kwargs) -> None:
+        """initializer method
+
+        Args:
+            name (str): handler name
+        """
+        super().__init__(name)
+
+        self.connection = None
+        self.is_connected = False
+        self._baseUrl = 'https://api.intercom.io'
+        args = kwargs.get('connection_data', {})
+        if 'access_token' in args:
+            access_token = args['access_token']
+        self._headers = {
+            "Accept": "application/json",
+            "Authorization": f"Bearer {access_token}"
+        }
+
+        _tables = [
+            Articles,
+        ]
+
+        for Table in _tables:
```

Comment:
Since we have only one Table let's just register without itteration

Reply:
Done